composer.json

```diff
@@ -27,6 +27,13 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.1"
+        "phpunit/phpunit": "^9.1",
+        "rector/rector": "^0.14.5"
```

Comment:
You don't need to add rector to composer (dev). There are other ways to run rector so that it does not become part of your dependencies. But it's okay for now I guess. We can remove it later when I add Quality Assurance tools.